docs/2gp_testing.rst

```diff
@@ -0,0 +1,123 @@
+Testing with Second-Generation Packaging
+========================================
+
+CumulusCI and the CumulusCI Flow make it easy to harness the power of second-generation
```

Comment:
```suggestion
CumulusCI makes it easy to harness the power of second-generation
```

(I think most of this is relevant regardless of what branch structure / git flow you're using.)